README.md

```diff
@@ -9,15 +9,18 @@ The SCP Ingest Pipeline is an ETL pipeline for single-cell RNA-seq data.
 
 # Prerequisites
 
-- Python 3.7+
+- Python 3.10
 - Google Cloud Platform project
 - Suitable service account (SA) and MongoDB VM in GCP. SA needs roles "Editor", "Genomics Pipelines Runner", and "Storage Object Admin". Broad Institute engineers: see instructions [here](https://github.com/broadinstitute/single_cell_portal_configs/tree/master/terraform-mongodb).
 - SAMtools, if using `ingest/make_toy_data.py`
 - Tabix, if using `ingest/genomes/genomes_pipeline.py`
 
 # Install
+
 ### Docker
+
 If on Apple silicon Mac (e.g. M1), do:
+
 ```
 scripts/docker-compose-setup.sh <PATH_TO_YOUR_VAULT_TOKEN> # E.g. ~/.github-token
```

Comment:
Thank you! I did discover the necessity of "-t" - it will be good to fix here as the error message when -t is absent didn't immediately lead me to the correct solution.

`0842d7d`